frontend_tests/puppeteer_lib/common.js

```diff
@@ -66,7 +66,8 @@ class CommonUtils {
                     "--no-sandbox",
                     "--disable-setuid-sandbox",
                 ],
-                defaultViewport: {width: 1280, height: 1024},
+                // TODO: Change defaultViewport to 1280x1024 when puppeteer fixes the window size issue with firefox.
```

Comment:
Can you link to an issue tracking the Puppeteer/firefox issue?

Reply:
@timabbott I have added the link to the issue.